CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+# 1.10.0
+# Added
+- New github Issue template
+- Signing support for rekeying to LogicSig/MultiSig account
```

Comment:
This is missing https://github.com/algorand/go-algorand-sdk/pull/233, which adds the new base64 asset fields

Reply:
Yup, I'm adding it now.

Reply:
#233 is added in Changelog and Release draft now.